requirements.txt

```diff
@@ -11,3 +11,4 @@ Django>=1.11
 freezegun>=0.3
 factory-boy>=2.8.1
 gprof2dot>=2016.10.13,<2017.09.19
+dealer>=2.0.5
```

Comment:
Checking out [dealer](https://github.com/klen/dealer), it seems dealer may be out of date.  Have you tried testing with django >= 1.10 (https://github.com/klen/dealer/issues/24)?

Reply:
I will take a look for an alternative to ``dealer``. I think the revision fuctionality is not that useful in production, at least we don't use it, we set ``SILKY_REVISION`` to a version number in ``settings.py``. But it is quite useful for benchmarking locally, for example for comparing a number of commits. If I can't find a better alternative I guess another option is to just remove this dependency and let users set to the revision number using whichever method they prefer (probably nice to have an example in the readme)

Reply:
I will check out the failing tests.